db/models/Shirt.js

```diff
@@ -21,6 +12,10 @@ const shirtSchema = new Schema({
   pic2: picSchema,
   pic3: picSchema,
   pic4: picSchema,
+  variation1: picSchema,
+  variation2: picSchema,
+  variation3: picSchema,
+  variation4: picSchema,
```

Comment:
It might make sense to refactor this to arrays instead of named properties (`variations: [picSchema]` instead of `variation1`, `variation2` etc.) in a later step...